src/routes/year/[year=year]/+page.svelte

```diff
@@ -142,15 +112,25 @@
 	}
 
 	function resetFilters() {
-		$selectedCreators = new Set($creators);
-		$selectedSeries = new Set($series);
-		$selectedEvents = new Set($events);
-		searchText = '';
-		month = months[0];
+		goto($page.url.pathname, { replaceState: true, keepFocus: true, noScroll: true });
+	}
+
+	let form: HTMLFormElement;
+	// todo: handle lack of support
+	const requestSubmit = () => form.requestSubmit();
```

Comment:
Might need to polyfill - only works on Safari 16+

Reply:
Though there's an issue on the polyfill that it doesn't actually work on Safari. Might be a niche case though (e.g. calling on submit button instead of form like we're doing here)

Reply:
Alternatively, it may be okay to force manual submit in that case

Reply:
We don't want manual submit because then debounced input would be annoying. "Tested" polyfill by removing the check for `requestSubmit` and seems to work okay (though obv not 100% valid test).